slipcover/slipcover.py

```diff
@@ -81,9 +82,10 @@ def matches(self, filename : Path):
 
 
 class Slipcover:
-    def __init__(self, collect_stats : bool = False, d_threshold = 50):
+    def __init__(self, collect_stats: bool = False, d_threshold: int = 50, branch: bool = False):
```

Comment:
What's the`d_threshold` for? Could use a more descriptive name, no doubt.

Reply:
Thanks. `d_miss_threshold` to trigger de-instrumentation.